Doc/c-api/intro.rst

```diff
@@ -46,7 +46,9 @@ Include Files
 =============
 
 All function, type and macro definitions needed to use the Python/C API are
-included in your code by the following line::
+included in your code by the following line:
+
+.. code:: c
```

Comment:
Is this block mishighlighted without this directive?

Reply:
When I had first opened the PR, I thought this would be needed, but it looks like it's fine in for this page since the header defines ``.. highlight:: c`` so any c syntax is automatically highlighted. I wasn't aware of that feature of reST at the time. Should the ``.. code::`` directive still be used here without the c part?